docs/maintainers/growing-your-project.md

```diff
@@ -0,0 +1,53 @@
+
+### How to identify new contributors and team members
+
+When it comes to planning the growth strategies for your open source project, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to their [OpenSauced profile](../docs/contributors/contributors-guide.md#your-profile) and checking their [dev card](../docs/features/dev-card.md). While you do this, ask yourself the following questions:
+
+- How often do they contribute to yours and other open source projects in general?
+- Are they invested in your project's success?
+
+Once you have your list of potential candidates, [create an a contributor insights page for your project](../docs/features/contributor-insights.md), and add your candidates to the page. From there, monitor their progress.
+
+::: tip
+
+To learn what to look for when monitoring your list of potential team members and contributors, check out our [Understanding Contributor Insights](../docs/maintainers/understanding-contrib-insights.md) guide.
+:::
+
+After you finished your observations, consider narrowing down your list to the contributors that suit your criteria the most. If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).
+
+### Recruiting team members and contributors
+
+When you're ready to recruit your team members, consider creating an [repository insights page](../docs/features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors you think suits your project, add them to a list, and monitor their progress.  
+
+:::tip
+Consider checking out our [Understanding Repository Insights Data](../docs/features/understanding-repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.
+:::
+
+Once you've finished your observations, recruit them. You can do this by going to their OpenSauced profiles and sending them a message via the contact information they provide.
+
+### Understanding team participation
+
+Once you have picked your official team members, it is crucial to monitor their participation to ensure that they are help the project run efficiently.  You can do this by using [the team sync feature](../docs/features/contributor-insights.md#how-to-create-a-contributor-insight-page).
```

Comment:
I wouldn't use crucial here. It can be "useful" or "helpful" to monitor your teams activity level by using a Contributor Insight page.